jsonrpc/eth_endpoint.go

```diff
@@ -188,6 +193,10 @@ func (e *Eth) GetTransactionReceipt(hash types.Hash) (interface{}, error) {
 
 // GetStorageAt returns the contract storage at the index position
 func (e *Eth) GetStorageAt(address types.Address, index types.Hash, number BlockNumber) (interface{}, error) {
+	//Set the block number to latest
+	if BlockNumber(0) == number {
+		number = LatestBlockNumber
+	}
```

Comment:
`number` will be overwritten to `LatestBlockNumber` when I give `0x0` to 3rd parameter. I think better to distinguish between (1) `0x0` is given (2) `number` is not given

Reply:
@munna0908 please take a look at @Kourin1996 's comment 🙏

Reply:
Fixed it, Here is the commit [link](https://github.com/0xPolygon/polygon-sdk/commit/7ed9279cd30c72299f4073dc9bf906cde80e4724)